mlflow/utils/rest_utils.py

```diff
@@ -30,11 +32,21 @@
 )
 from mlflow.utils.string_utils import strip_suffix
 
+
 RESOURCE_NON_EXISTENT = "RESOURCE_DOES_NOT_EXIST"
 _REST_API_PATH_PREFIX = "/api/2.0"
 _TRACE_REST_API_PATH_PREFIX = f"{_REST_API_PATH_PREFIX}/mlflow/traces"
 
 
+class _DatabricksSdkAPIErrorResponse(requests.Response):
```

Comment:
do we need to define `_DatabricksSdkAPIErrorResponse`? Can we directly use `requests.Response`?